content/tutorials/custom-zk-chain/10.index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Getting Started with ZK Chains
+description: Create and run your first ZK chain.
+---
+
+This tutorial shows you how to use the `zk_inception` CLI to run an Elastic Chain ecosystem and custom ZK chain locally:
+
+- You'll set up a local **Elastic Chain ecosystem**
+- You'll create a standard **ZK chain** and deploy a smart contract to it
+- You'll create a second ZK chain that uses a **custom  ERC20 base token**
+
+## Prerequisites
+
+- Make sure your machine satisfies the [system
+  requirements](https://github.com/matter-labs/era-compiler-solidity/tree/main#system-requirements).
+- If you aren't already familiar with deploying smart contracts on ZKsync Era, please  refer to the first section of the [quickstart tutorial](https://docs.zksync.io/build/start-coding/quick-start/deploy-your-first-contract).
+- For background on the Elastic Chain or ZK chains, read the [ZK chains](https://docs.zksync.io/zk-stack/concepts/zk-chains) section in our docs.
+- Install the dependencies for the `zksync-era` repo by following the instructions [here](https://github.com/matter-labs/zksync-era/blob/main/docs/guides/setup-dev.md)
+(you can skip the "Environment" section).
+
+::callout{icon="i-heroicons-light-bulb"}
+If you previously have `foundry-zksync` installed,
+reinstall the normal version of [foundry](https://book.getfoundry.sh/getting-started/installation) for this tutorial.
+::
+
+## Setting up the Elastic Chain ecosystem
+
+An Elastic Chain ecosystem is, in short, a system to manage multiple ZK chains.
+The vision for the Elastic Chain is an ever-expanding network of ZK rollups, secured by math and natively interoperable under a uniform, intuitive UX.
+New chains can be registered to the ecosystem to scale applications and communities as needed, making it "elastic".
+
+There are two components needed for running a ZK chain locally:
+
+1. An Elastic Chain ecosystem to manage different chains
+2. A chain deployed within the ecosystem
+
+To setup both of these components, use the `zk_inception` CLI.
+To install the CLI globally, run this command in your root folder.
+You can use `cargo` to install the CLI with the command below:
+
+```bash
+cargo install --git https://github.com/matter-labs/zksync-era/ \
+--locked zk_inception --force
+```
+
+::callout{icon="i-heroicons-light-bulb"}
+You can find a full reference for the `zk_inception` CLI in the [`zksync-era` repo](https://github.com/matter-labs/zksync-era/tree/main/zk_toolbox/crates/zk_inception).
+::
+
+Once you have that installed, the first step is to create a new ecosystem with the `ecosystem create` command.
+Move to a directory where you want your ecosystem folder to be, and run the command below to generate an ecosystem folder.
+
+```bash
+zk_inception ecosystem create
+```
+
+::callout{icon="i-heroicons-light-bulb"}
+Make sure to have the Docker daemon running.
```

Comment:
```suggestion
Make sure Docker is running on your machine.
```

I think it's better to refer to Docker via the Docker Desktop, even their guides focus on running via their app.